plugin.rb

```diff
@@ -1,9 +1,12 @@
 # name: discourse-oauth2-basic
 # about: Generic OAuth2 Plugin
-# version: 0.2
+# version: 0.3
 # authors: Robin Ward
 # url: https://github.com/discourse/discourse-oauth2-basic
 
+require 'uri'
+require 'net/http'
```

Comment:
These should not be required. Did you find you needed them for some reason?

(Rails includes both.)

Reply:
You're right, I'll remove them.